zilliqa/src/pool.rs

```diff
@@ -256,6 +256,27 @@ impl TransactionPool {
     pub fn size(&self) -> usize {
         self.transactions.len()
     }
+
+    pub fn has_txn_for_next_block(&self) -> bool {
```

Comment:
Maybe `has_txn_ready` would be better since it refers to the logic of the pool itself rather than blocks, which is something outside of this module. Maybe I'm nitpicking too much here though :)